admin/src/common/blocks/HeadingBlock.tsx

```diff
@@ -0,0 +1,103 @@
+import { BlockCategory, createCompositeBlock, createCompositeBlockSelectField } from "@comet/blocks-admin";
+import { createRichTextBlock } from "@comet/cms-admin";
+import { HeadingBlockData } from "@src/blocks.generated";
+import { FormattedMessage } from "react-intl";
+
+import { LinkBlock } from "./LinkBlock";
+
+const EyebrowRichTextBlock = createRichTextBlock({
+    link: LinkBlock,
+    rte: {
+        maxBlocks: 1,
+        supports: ["bold", "italic", "sub", "sup", "non-breaking-space", "soft-hyphen"],
+    },
+    minHeight: 0,
+});
+
+const HeadlineRichTextBlock = createRichTextBlock({
+    link: LinkBlock,
+    rte: {
+        maxBlocks: 1,
+        supports: [
+            "header-one",
+            "header-two",
+            "header-three",
+            "header-four",
+            "header-five",
+            "header-six",
+            "bold",
+            "italic",
+            "sub",
+            "sup",
+            "non-breaking-space",
+            "soft-hyphen",
+        ],
+        standardBlockType: "header-one",
+        blocktypeMap: {
+            "header-one": {
+                label: <FormattedMessage id="headingBlock.headline.size.600" defaultMessage="Size 600" />,
+            },
+            "header-two": {
+                label: <FormattedMessage id="headingBlock.headline.size.550" defaultMessage="Size 550" />,
+            },
+            "header-three": {
+                label: <FormattedMessage id="headingBlock.headline.size.500" defaultMessage="Size 500" />,
+            },
+            "header-four": {
+                label: <FormattedMessage id="headingBlock.headline.size.450" defaultMessage="Size 450" />,
+            },
+            "header-five": {
+                label: <FormattedMessage id="headingBlock.headline.size.400" defaultMessage="Size 400" />,
+            },
+            "header-six": {
+                label: <FormattedMessage id="headingBlock.headline.size.350" defaultMessage="Size 350" />,
```

Comment:
The IDs are not really consistent with the rest of the file, maybe change it to something like this: 

```suggestion
                label: <FormattedMessage id="headingBlock.size600" defaultMessage="Size 600" />,
            },
            "header-two": {
                label: <FormattedMessage id="headingBlock.size550" defaultMessage="Size 550" />,
            },
            "header-three": {
                label: <FormattedMessage id="headingBlock.size500" defaultMessage="Size 500" />,
            },
            "header-four": {
                label: <FormattedMessage id="headingBlock.size450" defaultMessage="Size 450" />,
            },
            "header-five": {
                label: <FormattedMessage id="headingBlock.size400" defaultMessage="Size 400" />,
            },
            "header-six": {
                label: <FormattedMessage id="headingBlock.size350" defaultMessage="Size 350" />,
```